bridge/src/main/java/com/livefront/bridge/Bridge.java

```diff
@@ -10,9 +10,9 @@
 
 public class Bridge {
 
-    private static BridgeDelegate sDelegate;
+    private static volatile BridgeDelegate sDelegate;
 
-    private static void checkInitialization() {
+    private static synchronized void checkInitialization() {
```

Comment:
Did you actually see this being an issue?

Reply:
I can't guarantee it is one, but there's a very odd bug reported and I figured it couldn't hurt to add this just in case it was some very strange threading issue.